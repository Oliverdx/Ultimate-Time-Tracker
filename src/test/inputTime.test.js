import React from 'react';
import renderer from 'react-test-renderer';
import InputTime from '../components/inputTime';

describe('InputTime component test', () => {

    it('Should render correctly', () => {
      const component = renderer.create(<InputTime updateTime={() => {}}/>).toJSON();
      expect(component).toMatchSnapshot();
    });
    
});