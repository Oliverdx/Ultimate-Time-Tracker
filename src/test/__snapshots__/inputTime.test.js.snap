// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputTime component test Should render correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <label>
    Data
    <input
      onChange={[Function]}
      type="date"
    />
  </label>
  <label>
    Início
    <input
      onChange={[Function]}
      type="time"
    />
  </label>
  <label>
    Saída
    <input
      onChange={[Function]}
      type="time"
    />
  </label>
  <label>
    Almoço
    <input
      onChange={[Function]}
      type="time"
    />
  </label>
  <label>
    Retorno
    <input
      onChange={[Function]}
      type="time"
    />
  </label>
  <input
    type="submit"
    value="Enviar"
  />
</form>
`;
